stetho/src/main/java/com/facebook/stetho/common/android/FragmentCompatUtil.java

```diff
@@ -26,13 +26,20 @@ public static Object findFragmentForView(View view) {
   @Nullable
   private static Object findFragmentForViewInActivity(Activity activity, View view) {
     FragmentCompat supportLib = FragmentCompat.getSupportLibInstance();
-    if (supportLib != null) {
+
+    // Try the support library version if it is present and the activity is FragmentActivity.
+    if (supportLib != null &&
+        supportLib.getFragmentActivityClass() == activity.getClass()) {
```

Comment:
This should be an isAssignableFrom() call, not a == comparison between Class<?>'s, right?